docs/production/export-and-import.md

```diff
@@ -460,11 +461,18 @@ s3_backups_bucket = # name of S3 backup bucket
 1. Run `/home/zulip/deployments/current/scripts/zulip-puppet-apply`.
 
 Daily full-database backups will be taken at 0200 UTC, and every WAL
-log will be written to S3 as it is saved by PostgreSQL; by default,
-this happens every 5 minutes, or every 1G, whichever happens first.
+archive will be written to S3 as it is saved by PostgreSQL; these are
+written every 16k of WAL log, which means that if there are periods of
+slow activity, it may be minutes before the backup is saved into S3 --
+see [`archive_timeout`][archive-timout] to set an upper bound on this.
+Because these are written every 16k, this may result in a large number
```

Comment:
This is fine but maybe we should include units with `16 KiB`, and possible also linkify "WAL" to something like https://www.postgresql.org/docs/current/wal-intro.html

Also worth noting that "WAL log" repeats "log" twice; I don't think deleting "log" would be an improvement, but I suppose we could consider something like "every 16KiB of [write-ahead log (WAL)]()"

Reply:
Technically, they're "WAL archives," so I'll switch to that.

Reply:
I don't think I want to use "continuous" because that implies that there is continuity to them, in that you will hardly lose any data -- which is not the case, as one may lose seconds or minutes of data.  Which is perhaps an impression I'm already worsening by calling them "streaming backups."